modules/ROOT/pages/microsoft/microsoft-365-ops-connector-reference.adoc

```diff
@@ -332,36 +322,32 @@ execute any operation for that user without doing the authorization dance again.
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS | A time unit that qualifies the maxIdleTime attribute. |  |
+** DAYS | A time unit that qualifies the *Max Idle Time* attribute. |  | 
 |===
 
 [[repeatable-in-memory-iterable]]
 === Repeatable In Memory Iterable
 
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Initial Buffer Size a| Number | The number of instances that can initially stay in memory to consume a stream and provide random access to it. If the stream contains more data than can fit into this buffer, the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. Default value is 100 instances. |  |
-| Buffer Size Increment a| Number | How much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. Default value is 100 instances. |  |
-| Max Buffer Size a| Number | The maximum amount of memory to use. If more memory is needed, the STREAM_MAXIMUM_SIZE_EXCEEDED error occurs. A value lower or equal to zero means no limit. |  |
+| Initial Buffer Size a| Number | The amount of instances that is initially be allowed to be kept in memory to consume the stream and provide random access to it. If the stream contains more data than can fit into this buffer, then the buffer expands according to the *Buffer Size Increment* attribute, with an upper limit of maxInMemorySize. Default value is 100 instances. |  | 
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, and that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. Default value is 100 instances. |  | 
+| Max Buffer Size a| Number | The maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower than or equal to zero means no limit. |  | 
 |===
 
 [[repeatable-file-store-iterable]]
 === Repeatable File Store Iterable
 
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Max In Memory Size a| Number | The maximum amount of instances to keep in memory. If more are required, the connector starts to buffer the content on disk. |  |
+| Max In Memory Size a| Number | The maximum amount of instances that will be kept in memory. If more than that is required, then it will start to buffer the content on disk. |  | 
```

Comment:
```suggestion
| Max In Memory Size a| Number | The maximum number of instances that will be kept in memory. If more than that is required, then it will start to buffer the content on disk. |  | 
```